src/sleep.cc

```diff
@@ -14,7 +14,7 @@
 
 #include "src/sleep.h"
 
-#include "src/platform.h"
+#include "amber/platform.h"
```

Comment:
I don't think we want to push this into the public API. I'd be more inclined to duplicate it into samples/ if that's where you need it.

Reply:
Agreed.